index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <title>Setup and mobile version</title>
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+  <body>
+    <header>
+      <div class="logo">Mucyo</div>
+      <div class="menu-toggle">
+        <i class="fa fa-bars"></i>
+      </div>
+    </header>
+
+    <section class="headline-page">
+      <div class="introduction">
+        <h1 class="intro">Hey There. I'm Mucyo</h1>
+        <h2 class="headline-page-h2">I'am a Software Developer</h2>
+        <p class="text-1">
+          I can help you build a product, feature or website Look through some
+          of my work and experience! if you like what you see and have a project
+          you need coded, don't hestiate to contacta me.
+        </p>
+      </div>
+      <ul class="social-icons">
+        <li class="fa-size">
+          <a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a>
+        </li>
+        <li class="fa-size">
+          <a href="#"><i class="fa fa-linkedin" aria-hidden="true"></i></a>
+        </li>
+        <li class="fa-size">
+          <a href="#"><i class="fa fa-medium" aria-hidden="true"></i></a>
+        </li>
+        <li class="fa-size">
+          <a href="#"><i class="fa fa-github" aria-hidden="true"></i></a>
+        </li>
+        <li class="fa-size">
+          <a href="#"><i class="fa fa-angellist" aria-hidden="true"></i></a>
+        </li>
+      </ul>
+    </section>
+    <div class="middle_container">
+      <img src="images/Img.png" alt="image" class="middle-image" />
+      <div class="main_block">
+        <div class="post">Multi-Post Stories</div>
+        <div class="text_post">
+          <p>
```

Comment:
![image](https://user-images.githubusercontent.com/60164239/129298727-83ed477c-7435-4e24-92fa-5de9f844f3fb.png)

- [ ] The title (e.g., "My Recent Works") uses a header tag `hx`, currently is missing as you can see in the red color.
- [ ] Add some padding to center more your content according to the Figma design you selected, as you can see in the yellow color.